components/ocmcli/Dockerfile

```diff
@@ -1,21 +1,18 @@
 # This Dockerfile is used by `make` via the Makefile
 
-ARG ALPINE_VERSION="3.20.2"
-FROM --platform=$BUILDPLATFORM alpine:${ALPINE_VERSION}
-
-# Create group and user
-ARG UID=1000
-ARG GID=1000
-RUN addgroup -g "${GID}" ocmGroup && adduser -u "${UID}" ocmUser -G ocmGroup -D
+# debian12:nonroot
```

Comment:
I guess thats the default/only user of that image? I would expect an explanation for this comment or delete it

Reply:
The "nonroot" images are configured to default to executing as a normal user account called "nonroot", with user ID 65532. That is the only difference. All the images contain configuration for the user accounts root, nobody, and nonroot.

Reply:
But do we need the comment?

Reply:
Explicitly added it via the image to avoid needing the comment. Please double check